src/main/java/net/sf/jabref/wizard/text/gui/TextInputDialog.java

```diff
@@ -479,9 +482,13 @@ private void updateSourceView() {
     }
 
     private String[] getAllFields() {
-        List<String> texFields = EntryTypes.getType(entry.getType(), frame.getCurrentBasePanel().getBibDatabaseContext().getMode()).getAllFields();
+        List<String> texFields = new ArrayList<>();
+        Optional<EntryType> type = EntryTypes.getType(entry.getType(),
+                frame.getCurrentBasePanel().getBibDatabaseContext().getMode());
+        if (type.isPresent()) {
+            texFields = type.get().getAllFields();
```

Comment:
Good point! I edited the PR accordingly. So you can just remove the corresponding statement in #760 and both PRs should be merged in without problems.